$OpenBSD$
Cannot assign with class initializer.
Move it to the one method that it's used.

Index: test/test_va_api_fixture.h
--- test/test_va_api_fixture.h.orig
+++ test/test_va_api_fixture.h
@@ -102,7 +102,6 @@ class VAAPIFixture : public ::testing::Test { (protect
 private:
     char *m_restoreDriverName;
     int m_drmHandle;
-    const std::string drmDevicePaths[2];
 
     VAConfigID m_configID;
     VAContextID m_contextID;
